app_legacy/Helpers/render/layout.php

```diff
@@ -105,16 +107,28 @@ function RenderToolbar(): void
     echo "</ul>";
 
     echo "<ul>";
+    echo "<li class='dropdown-header'>Miscellaneous</li>";
     echo "<li><a href='/gameList.php'>All Games</a></li>";
     // echo "<li><a href='/popularGames.php'>Most Played</a></li>";
     echo "<li><a href='/setRequestList.php'>Most Requested</a></li>";
+    echo "<li class='dropdown-header'>Hubs</li>";
+    echo "<li><a href='/gameList.php?s=6&c=100&f=1'>Hub List</a></li>";
+    echo "<li><a href='/game/6914'>Central Hub</a></li>";
+    echo "<li><a href='/game/9553'>Genre & Subgenre Hub</a></li>";
+    echo "<li><a href='/game/5771'>Series Hub</a></li>";
+    echo "<li><a href='/game/3105'>Community Events</a></li>";
+    echo "<li><a href='/game/3273'>Developer Events</a></li>";
```

Comment:
Should `Community Events` and `Developer Events` also specify Hub? i.e. `Community Events Hub`?

Reply:
Specified Hub for the events, but I'm wondering if any of them even need it since they're already under the Hub category 🤔